spec/spec_helper.rb

```diff
@@ -1,3 +1,44 @@
+# The tests run against a mongo server or a replica set which is
```

Comment:
This technically makes it sound like sharded clusters aren't supported; it's probably best to just say "The tests run run against a MongoDB cluster..."

Reply:
Changed.